.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
                         coverage: 'pcov'
                     -   php: '8.3'
                         os: ubuntu-latest
-                        symfony: '^6.0'
+                        symfony: '^7.0'
```

Comment:
actually, flex silently failing and not applying this rule. we can merge as-is, and when Sf7 is out, it will start applying the change